skins/cat17/src/app/lib/view_handler/display_payment_summary.js

```diff
@@ -1,202 +1,193 @@
-
-        this.setSummaryIcon(this.memberShipTypeIconElement, formContent.membershipType, this.memberShipTypeIcon);
-
-        this.memberShipTypeTextElement.text(this.memberShipTypeText[formContent.membershipType]);
-      }
-    },
-    capitalize: function (s) {
-      return s[0].toUpperCase() + s.slice(1);
-    },
-    formatPaymentInterval: function ( paymentIntervalInMonths ) {
-      return this.intervalTranslations[ paymentIntervalInMonths ];
-    },
-    formatPaymentType: function ( paymentType ) {
-      return paymentType=="" ? String("Zahlung noch nicht ausgewählt.") : this.paymentTypeTranslations[ paymentType ];
-    },
-    getAddressSummaryContent: function (formContent) {
-      if (formContent.addressType === "person") {
-		  // TODO Refactor into something more readable.
-        return (
-          formContent.firstName && formContent.lastName ?
-            (formContent.salutation && formContent.title !== 'kein-title' && formContent.title !== 'vtitle' && formContent.salutation != 'anrede' ?
-              this.capitalize(formContent.salutation) : "") + " " +
-            (formContent.title && formContent.title !== 'kein-title' && formContent.title !== 'vtitle' ?
-              this.capitalize(formContent.title) : "") + " " +
-            formContent.firstName + " " + formContent.lastName + "<br />"
-            : ""
-        ) +
-        (formContent.street ? formContent.street + "<br />" : "") +
-        (formContent.postcode && formContent.city ? formContent.postcode + " " + formContent.city + "<br />" : "") +
-        ( formContent.country ? this.countriesDictionary[formContent.country] + "<br />" : "") +
-        formContent.email;
-      }
-      else if (formContent.addressType === 'firma') {
-        return (formContent.companyName ? formContent.companyName + "<br />" : "") +
-        (formContent.contactPerson ? formContent.contactPerson + "<br />" : "") +
-        (formContent.street ? formContent.street + "<br />" : "") +
-        (formContent.postcode && formContent.city ? formContent.postcode + " " + formContent.city + "<br />" : "") +
-        ( formContent.country ? this.countriesDictionary[formContent.country] + "<br />" : "") +
-        formContent.email;
-      }
-
-      return "";
-    },
-    setSummaryIcon: function (elements, value, iconsDictionary) {
-      elements.removeClass('icon-error');
-      if (elements.length && elements.get(0).className.split(' ').length > 1) {
-        elements.removeClass(elements.get(0).className.split(' ').pop());
-      }
-		// @fixme Don't use jQuery here. Instead, interact with the store values and the elements.
-      if (iconsDictionary[value] === undefined) {
-        elements.each(function() {
-          if ($(this).data('display-error') === undefined) {
-            $(this).addClass('icon-error');
-          }
-        });
-      }
-      else {
-        elements.addClass( iconsDictionary[value] );
-      }
-    }
-  };
+	DATA_EMPTY_TEXT = 'empty-text',
```

Comment:
When I first saw `DATA_EMPTY_TEXT` I was a bit confused and thought it to be a translation key. I think we should make it clearer that these constants have something to do with the DOM. Maybe with custom functions that use the constants, maybe with a private `Element` class or just by adding a comment.